ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java

```diff
@@ -3861,8 +3861,8 @@ public synchronized void start( TransExecutionConfiguration executionConfigurati
           // memory
           // To be able to completely test this, we need to run it as we would normally do in pan
           //
-          trans = transMeta.getTransSupplier().get( transMeta, log, this::createLegacyTrans );
-
+          trans = DefaultTransManager.getInstance().createTrans( transMeta, log,
+            executionConfiguration.getRunConfiguration(), this::createLegacyTrans );
```

Comment:
see comment https://github.com/pentaho/pentaho-kettle/pull/9966/commits/50e4e551c96a5479288ba79e1ebb32d86212d526#r2069538322 for use of `this::createLegacyTrans`

for context `TransGraph#createLegacyTrans`

```
  private Trans createLegacyTrans() {
    try {
      return new Trans( transMeta, spoon.rep, transMeta.getName(),
        transMeta.getRepositoryDirectory().getPath(),
        transMeta.getFilename(), transMeta);
    } catch ( KettleException e ) {
      throw new RuntimeException( e );
    }
  }
```

Reply:
addressed